alembic/versions/684501e372d2_adding_oauth.py

```diff
@@ -0,0 +1,82 @@
+"""
+Adding oauth.
+
+Revision ID: 684501e372d2
+Revises: 65167edbb0de
+Create Date: 2023-05-17 20:39:21.658338
+
+"""
+import fastapi_users_db_sqlalchemy as fastapi_users_db_sqlalchemy
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "684501e372d2"
+down_revision = "65167edbb0de"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    """Upgrade the schema to this version."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "user",
+        sa.Column("id", fastapi_users_db_sqlalchemy.generics.GUID(), nullable=False),
+        sa.Column("email", sa.String(length=320), nullable=False),
+        sa.Column("hashed_password", sa.String(length=1024), nullable=False),
+        sa.Column("is_active", sa.Boolean(), nullable=False),
+        sa.Column("is_superuser", sa.Boolean(), nullable=False),
+        sa.Column("is_verified", sa.Boolean(), nullable=False),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_index(op.f("ix_user_email"), "user", ["email"], unique=True)
+    op.create_table(
+        "oauth_account",
+        sa.Column("id", fastapi_users_db_sqlalchemy.generics.GUID(), nullable=False),
+        sa.Column(
+            "user_id", fastapi_users_db_sqlalchemy.generics.GUID(), nullable=False
+        ),
+        sa.Column("oauth_name", sa.String(length=100), nullable=False),
+        sa.Column("access_token", sa.String(length=1024), nullable=False),
+        sa.Column("expires_at", sa.Integer(), nullable=True),
+        sa.Column("refresh_token", sa.String(length=1024), nullable=True),
+        sa.Column("account_id", sa.String(length=320), nullable=False),
+        sa.Column("account_email", sa.String(length=320), nullable=False),
+        sa.ForeignKeyConstraint(["user_id"], ["user.id"], ondelete="cascade"),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_index(
+        op.f("ix_oauth_account_account_id"),
+        "oauth_account",
+        ["account_id"],
+        unique=False,
+    )
+    op.create_index(
+        op.f("ix_oauth_account_oauth_name"),
+        "oauth_account",
+        ["oauth_name"],
+        unique=False,
+    )
+    op.drop_table("users")
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
+    """Downgrade the schema to this version."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "users",
+        sa.Column("id", sa.VARCHAR(), autoincrement=False, nullable=False),
```

Comment:
Why not use `fastapi_users_db_sqlalchemy.generics.GUID()` as we did in the other table?

Reply:
I'm not sure! Again -- this is code generated by alembic bast on the FastAPI-User models